wal/wal_test.go

```diff
@@ -611,7 +611,7 @@ func TestRestartCreateWal(t *testing.T) {
 
 	// make temporary directory so it looks like initialization is interrupted
 	tmpdir := path.Clean(p) + ".tmp"
-	if err = os.Mkdir(tmpdir, fileutil.PrivateDirMode); err != nil {
+	if err = fileutil.TouchDirAll(tmpdir); err != nil {
```

Comment:
Right, ioutil.TempDir already creates directory.

Just reverted this part.

Thanks.